corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -184,23 +184,26 @@ var COMMCAREHQ = (function () {
     'use strict';
     return {
         icons: {
-            GRIP:   'icon-resize-vertical icon-blue',
-            ADD:    'icon-plus icon-blue',
-            COPY:   'icon-copy icon-blue',
-            DELETE: 'icon-remove icon-blue',
-            PAPERCLIP: 'icon-paper-clip'
+            GRIP:   'fa fa-sort icon-resize-vertical icon-blue',
+            ADD:    'fa fa-plus icon-plus icon-blue',
+            COPY:   'fa fa-copy icon-copy icon-blue',
+            DELETE: 'fa fa-remove icon-remove icon-blue',
+            PAPERCLIP: 'fa fa-paperclip icon-paper-clip'
```

Comment:
yes, because the old styles are still in there. when used on an older template, those styles will take over